lib/citgm.js

```diff
@@ -120,11 +121,11 @@ Tester.prototype.run = function() {
         self.emit('fail', err);
         payload.error = err;
       }
-      if (self.testOutput !== '') {
-        payload.testOutput += self.testOutput + '\n';
+      if (self.testOutput.length) {
+        payload.testOutput += self.testOutput.toString() + '\n';
       }
-      if (self.testError !== '') {
-        payload.testOutput += self.testError + '\n';
+      if (self.testError.length) {
+        payload.testOutput += self.testError.toString() + '\n';
```

Comment:
What's the impetus towards doing the `toString` later in the application? I agree it seems like a bit of a hack to be passing massive strings through the functions, but I'm not clear on the benefit of moving BufferLists instead.